modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/BaseDistributionZoneManagerTest.java

```diff
@@ -177,6 +188,14 @@ protected void createZone(
             @Nullable Integer dataNodesAutoAdjustScaleDown,
             @Nullable String filter
     ) {
+        DistributionZonesTestUtil.createZone(
```

Comment:
These two calls to `createZone()` look almost identically, it's hard to distinguish them. Would it make sense to rename both methods to make them look like `createZoneInCatalog()` and `createZoneInManager()`?

Reply:
I don't think it makes sense as it will be removed in the neighboring PR [(ignite-3/pull/2415)](https://github.com/apache/ignite-3/pull/2415) in the near future.